docs/docs/guides/deploy/kubernetes.mdx

```diff
@@ -12,23 +12,29 @@ For running an easily testable, insecure, non-HA ZITADEL instance, run the follo
 
 ## Helm
 
-### Add the helm repository
+### Add the helm repositories for CockroachDB and ZITADEL
 
 ```bash
+helm repo add cockroachdb https://charts.cockroachdb.com/
 helm repo add zitadel https://charts.zitadel.com
 ```
 
-### Install an insecure zitadel release that works with localhost
+### Install an insecure cockroachdb and zitadel release that works with localhost
 
 ```bash
+# CockroachDB
+helm install --namespace zitadel --create-namespace crdb cockroachdb/cockroachdb \
+  --set fullnameOverride=crdb \
+  --set single-node=true \
+  --set statefulset.replicas=1
+
+# ZITADEL
 helm install --namespace zitadel --create-namespace my-zitadel zitadel/zitadel \
   --set zitadel.masterkey="MasterkeyNeedsToHave32Characters" \
   --set zitadel.configmapConfig.ExternalSecure=false \
   --set zitadel.configmapConfig.TLS.Enabled=false \
   --set zitadel.secretConfig.Database.cockroach.User.Password="a-zitadel-db-user-password" \
   --set replicaCount=1 \
```

Comment:
this should be deleted, otherwise the command won't execute

Reply:
marked the wrong lines, sry. I mean the last char at 29

Reply:
thx